kuma/users/tests/test_tasks.py

```diff
@@ -30,49 +40,62 @@ def test_default_language_email(self):
     def test_dont_send_untranslated_language_email(self,
                                                    strings_are_translated):
         strings_are_translated.return_value = False
-        u = User.objects.get(username='testuser')
-        send_welcome_email(u.pk, 'tlh')  # Qapla'
+        testuser = User.objects.get(username='testuser')
+        send_welcome_email(testuser.pk, 'tlh')  # Qapla'
         eq_([], mail.outbox)
 
-        send_welcome_email(u.pk, 'de')  # Servus
+        send_welcome_email(testuser.pk, 'de')  # Servus
         eq_(1, len(mail.outbox))
 
     def test_welcome_mail_for_verified_email(self):
         Switch.objects.get_or_create(name='welcome_email', active=True)
-        u = user(username='welcome',
-                 email='welcome@tester.com',
-                 password='welcome',
-                 save=True)
-        request = RequestFactory().get('/')
-        request.locale = 'en-US'
-        user_signed_up.send(sender=u.__class__, request=request, user=u)
+        testuser = user(username='welcome', email='welcome@tester.com',
+                        password='welcome', save=True)
+        request = self.setup_request_for_messages()
+        self.get_messages(request)
```

Comment:
We still need `get_messages` because it has the FIXME/HACK to put `FallbackStorage` messages onto `request._messages`, which the code under test needs.